desk/sur/chat.hoon

```diff
@@ -1,4 +1,5 @@
 /-  d=channels
+/-  c2=chat-2, c3=chat-3, c4=chat-4, c5=chat-5
```

Comment:
We've talked about this previously, but for the record: I don't like these dependencies just to re-export them. If you need one, import the relevant file.